client/my-sites/checkout/composite-checkout/components/refund-policies.tsx

```diff
@@ -222,6 +266,13 @@ function RefundPolicyItem( { refundPolicy }: { refundPolicy: RefundPolicy } ) {
 			);
 			break;
 
+		case RefundPolicy.GiftYearlyPurchase:
+			text = translate(
+				'You understand that gift {{refundsSupportPage}}refunds{{/refundsSupportPage}} are limited to 14 days after purchase for products with yearly subscriptions.',
```

Comment:
If you simplify, let's make this a little more generic (and change the switch case):

`You understand that {{refundsSupportPage}}refunds{{/refundsSupportPage}} are limited to 14 days after purchase for non-domain gift subscriptions.`

Reply:
You can ignore my comment: p1669821415337469/1669748340.840929-slack-C0117V2PCAE